src/stores/AltStore.js

```diff
@@ -7,7 +7,7 @@ class AltStore {
     this.dispatches = {}
     this.stores = {}
     this.selectedAlt = null
-    this.logDispatches = true
+    this.logDispatches = false
```

Comment:
we want this to be true though, in an action we have a short circuit if we aren't logging dispatches.